starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,61 @@
+'use strict';
+
 // Names and Input
+var hacker1 = "dave";
+console.log("the drivers name is " + hacker1);
+
+let hacker2 = "hello";
```

Comment:
Two things: 
- try to keep your code consisten, you use everywhere var and here let...
- it isn't mandatory to declare the variable before and then change it. You can do : 
```
var hacker2 = prompt('.....')
console.log('....' + hacker2)
```